src/styles/index.scss

```diff
@@ -1,3 +1,32 @@
+@import '../styles/blocks/utils/variables';
+@import '../styles/blocks/header';
+@import '../styles/blocks/card';
+@import '../styles/blocks/stars';
```

Comment:
generally try adding the import files in a completely separate file, for example, use the imports on index and selectors for body and * in something like globals.css